.github/workflows/master.yml

```diff
@@ -8,45 +8,43 @@ jobs:
   assemble:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - name: set up JDK 11
-      uses: actions/setup-java@v3
-      with:
-        distribution: 'adopt'
-        java-version: 11
-    - name: run assemble
-      run: ./gradlew assemble
+      - uses: actions/checkout@v3
+      - name: set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: run assemble
+        run: ./gradlew assemble
 
   test:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - name: set up JDK 11
-      uses: actions/setup-java@v3
-      with:
-        distribution: 'adopt'
-        java-version: 11
-    - name: run testDebugUnitTest
-      run: ./gradlew testDebugUnitTest
-    - name: run jacocoTestReport
-      run: ./gradlew jacocoTestReport
-    - uses: codecov/codecov-action@v3
+      - uses: actions/checkout@v3
+      - name: set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: run kover
```

Comment:
These two lines are the only changes, everything else was VSCode auto formatting